packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -91,4 +103,31 @@ describe('Google Analytics 4 events tests', () => {
       }
     });
   });
-});
+})
+
+describe('Google Analytics 4 Identify events tests', () => {
+  let ga4;
+  beforeEach(() => {
+    ga4 = new GA4(
+      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', sendUserTraitsAsPartOfInIt: true, piiPropertiesToIgnore: [{ piiProperty: {} }] },
```

Comment:
Add test for `piiProperty` not present, and present with multiple values

Reply:
this scenario got covered in `filterUserTraits` function tests